src/Data/Nat/Properties.agda

```diff
@@ -214,14 +217,9 @@ n≤0⇒n≡0 z≤n = refl
 <-transˡ : Trans _<_ _≤_ _<_
 <-transˡ (s≤s m≤n) (s≤s n≤o) = s≤s (≤-trans m≤n n≤o)
 
-<-cmp : Trichotomous _≡_ _<_
-<-cmp zero    zero    = tri≈ (λ())     refl  (λ())
-<-cmp zero    (suc n) = tri< (s≤s z≤n) (λ()) (λ())
-<-cmp (suc m) zero    = tri> (λ())     (λ()) (s≤s z≤n)
-<-cmp (suc m) (suc n) with <-cmp m n
-... | tri< ≤ ≢ ≱ = tri< (s≤s ≤)      (≢ ∘ suc-injective) (≱ ∘ ≤-pred)
-... | tri≈ ≰ ≡ ≱ = tri≈ (≰ ∘ ≤-pred) (cong suc ≡)        (≱ ∘ ≤-pred)
-... | tri> ≰ ≢ ≥ = tri> (≰ ∘ ≤-pred) (≢ ∘ suc-injective) (s≤s ≥)
+
+-- <-cmp  was here ***
+
```

Comment:
We don't need this information.

Reply:
Yes, the line No 221 needs to be removed. I have forgotten to do this.